client/my-sites/plan-price/index.jsx

```diff
@@ -1,6 +1,7 @@
 import { getCurrencyObject } from '@automattic/format-currency';
 import classNames from 'classnames';
 import { localize } from 'i18n-calypso';
+import { isNumber } from 'lodash';
```

Comment:
Is there any chance we could remove the usage of Lodash `isNumber`? This is currently the only use of this method and we got rid of all of them a while ago in #52156. Let me know if you need any help 🙌